stable/yugaware/templates/statefulset.yaml

```
@@ -72,7 +75,14 @@ spec:
             items:
               - key: prometheus.yml
                 path: prometheus.yml
-
+        {{- if .Values.securityContext.enabled }}
+        - name: init-container-script
+          configMap:
+            name: {{ .Release.Name }}-yugaware-init-container
```

```suggestion
            name: {{ .Release.Name }}-yugaware-init
```
Mostly to use less number of characters.